trace-dispatcher/src/Cardano/Logging/Utils.hs

```diff
@@ -44,15 +44,16 @@ runInLoop action localSocket prevDelayInSecs =
 uncurry3 :: (a -> b -> c -> d) -> (a, b, c) -> d
 uncurry3 f (a,b,c) = f a b c
 
+-- | map over second element of a tuple
 mapSnd :: (a -> b) -> (c, a) -> (c, b)
```

Comment:
Just a veeeeeeery minor thing: There is `Data.Bifunctor.second` in `base`....

Reply:
Removed